include/sentry.h

```diff
@@ -505,15 +504,21 @@ typedef struct sentry_options_s sentry_options_t;
  * This represents an interface for user-defined transports.
  *
  * Transports are responsible for sending envelopes to sentry and are the last
- * step in the event pipeline. A transport has the following hooks, all of which
+ * step in the event pipeline.
+ *
+ * Envelopes will be submitted to the transport in a _fire and forget_ fashion,
+ * and the transport must send those envelopes _in order_.
```

Comment:
Envelopes have to be sent in order now? So the transport can't send envelopes in parallel then.
This should be noted in the CHANGELOG as a breaking change as well.

Reply:
this has been the case ever since release health, since that relies on getting the session updates in order. But sure, I can add it to the changelog as well.